### PR TITLE
ami: update Grafana to 9.0.7 (#4218)

### LINKED ISSUE #4218: Update Grafana in AMI / Examples
### Description
Grafana has fixed two important CVEs https://grafana.com/blog/2020/06/03/grafana-6.7.4-and-7.0.2-released-with-important-security-fix/ 

https://github.com/gravitational/teleport/blob/c33dde526311c54ff9585dfb80586b6695561bc4/assets/aws/Makefile#L44 We should bump to 6.7.4 for an important security fix.

### PR DESCRIPTION
**Pull request by hugoShaka**
Fixes #4218

The PR:
- Bumps the Grafana version
- Fixes the packer script failing when `DEBUG` is unset
- Makes Nginx propagate the Host header required since https://grafana.com/blog/2022/02/08/grafana-7.5.15-and-8.3.5-released-with-moderate-severity-security-fixes/
- Makes Nginx proxy websocket calls properly (kinda necessary since grafana 8)
- Retab Nginx conf files that were a mix of tabs and spaces
- Makes Terraform compute s3 files' md5 to detect and apply changes

### CODE REVIEW

**Comment by hugoShaka** [comment]
Note: I'm not sure if this should be backported to v9/v10.

Pros:
- this includes a bunch of Grafana security fixes
- The terraform md5 computation is a painful gotcha
- Our monitor instances are mainly stateless and this has no interaction with the product
Cons:
- this would break the semver, biggets change is the grafana renderer removal

File: examples/aws/terraform/ha-autoscale-cluster/s3.tf [hunk 1]
```diff
@@ -16,6 +16,7 @@ resource "aws_s3_bucket_object" "grafana_teleport_dashboard" {
   bucket = aws_s3_bucket.certs.bucket
   key    = "health-dashboard.json"
   source = "./assets/health-dashboard.json"
+  etag    = filemd5("./assets/health-dashboard.json")
```
Comment (r0mant) [inline examples/aws/terraform/ha-autoscale-cluster/s3.tf:19]:
```suggestion
  etag   = filemd5("./assets/health-dashboard.json")
```

File: examples/aws/terraform/ha-autoscale-cluster/s3.tf [hunk 2]
```diff
@@ -24,6 +25,7 @@ resource "aws_s3_bucket_object" "grafana_teleport_nginx" {
   key    = "grafana-nginx.conf"
   source = "./assets/grafana-nginx.conf"
   count  = var.use_acm ? 0 : 1
+  etag    = filemd5("./assets/grafana-nginx.conf")
```
Comment (r0mant) [inline examples/aws/terraform/ha-autoscale-cluster/s3.tf:28]:
```suggestion
  etag   = filemd5("./assets/grafana-nginx.conf")
```

File: examples/aws/terraform/ha-autoscale-cluster/s3.tf [hunk 3]
```diff
@@ -32,5 +34,6 @@ resource "aws_s3_bucket_object" "grafana_teleport_nginx_acm" {
   key    = "grafana-nginx.conf"
   source = "./assets/grafana-nginx-acm.conf"
   count  = var.use_acm ? 1 : 0
+  etag    = filemd5("./assets/grafana-nginx-acm.conf")
```
Comment (r0mant) [inline examples/aws/terraform/ha-autoscale-cluster/s3.tf:37]:
```suggestion
  etag   = filemd5("./assets/grafana-nginx-acm.conf")
```

**Review by r0mant — APPROVED** [review]
Should be ok to just keep this update for master branch.

**Review by ravicious — APPROVED** [review]

**Pull request merged**